guides/advanced_guides/settings.md

```diff
@@ -0,0 +1,251 @@
+  --data '{
+      "stopWords": [
+          "the",
+          "a",
+          "an"
+      ]
+  }'
+```
+
+In the example above, `the`, `a` and `an` are ignored by the sorting algorithm if they are present in search queries.
+
+Suppose you would like to search `the mask` in a movie database. Since `the` is ignored during search, MeiliSearch will look for every movie containing `mask` and not the millions ones containing `the`. `the` is a less relevant term than `mask` and also a very frequent word in English. By adding `the` to the stop words list, MeiliSearch will ignore this word, and thus be faster to answer without losing in relevancy.
+
+## Ranking rules
+
+Built-in ranking rules that **ensure relevancy in search results**. Ranking rules are applied in a default order which can be changed in the settings. You can add or remove rules and change their order of importance.
+
+`rankingRules=[<String>, <String>, ...]`
+
+- `[<String>, <String>, ...]` (Array of strings, see default value below)
+
+  An array of strings that contains the ranking rules sorted by order of importance (arranged from the most important rule to the least important rule).
+
+Default value:
+
+```json
+["typo", "words", "proximity", "attribute", "wordsPosition", "exactness"]
+```
+
+By default, rules are executed in the following order:
+
+#### 1. Typo
+
+Results are sorted by **increasing number of typos**: find documents that match query terms with fewer typos first.
+
+#### 2. Words
+
+Results are sorted by **decreasing number of matched query terms** in each matching document: find documents that contain more occurrences of the query terms first.
+
+#### 3. Proximity
+
+Results are sorted by **increasing distance between matched query terms**: find documents that contain more query terms found close together (close proximity between two query terms) and appearing in the original order specified in the query string first.
+
+#### 4. Attribute
+
+Results are sorted according to **[the order of importance of the attributes](/guides/main_concepts/relevancy.md#attributes-importance)**: find documents that contain query terms in more important attributes first.
+
+#### 5. Words Position
+
+Results are sorted by **the position of the query words in the attributes**: find documents that contain query terms earlier in their attributes first.
+
+#### 6. Exactness
+
+Results are sorted by **the similarity of the matched words with the query words**: find documents that contain exactly the same terms as the ones queried first.
+
+[Learn more about ranking rules](/guides/main_concepts/relevancy.md)
+
+#### Example
+
+Add settings:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "rankingRules": [
+          "typo",
+          "words",
+          "proximity",
+          "attribute",
+          "wordsPosition",
+          "exactness",
+          "asc(release_date)",
+          "desc(rank)"
+      ]
+  }'
+```
+
```

Comment:
Could you add a explaination of the above example ?
For example, this means that documents will first be sorted by number of typos and then if to many documents have the equal number of typos we will apply the words rules on it and so one until we have limited the number of documents to the number asked (default: 20)